docs/querying/lookups.md

```diff
@@ -62,6 +62,12 @@ SELECT
 FROM sales
 GROUP BY 1
 ```
+The lookup function also accepts the 3rd argument called `replaceMissingValueWith` as a constant string. If your value is missing a lookup for the queried key, the lookup function returns the result value from `replaceMissingValueWith`
+For example:
+```
+LOOKUP(store, 'store_to_country', 'NA')
+```
+If value is missing from `store_to_country` lookup for given key 'store' then it will return `NA`.
```

Comment:
you should probably indicate that this optional 3rd argument must be a constant.

`math-expr.md`, `sql-functions.md`, and `sql-scalar.md` should also be updated since they all document this function signature

Reply:
fixed